app/_hub/kong-inc/request-transformer-advanced/index.md

```diff
@@ -163,7 +163,7 @@ $ curl -X POST http://localhost:8001/apis \
 
 Enable the ‘request-transformer-advanced’ plugin to add a new header `x-consumer-id`
 and its value is being set with the value sent with header `x-user-id` or
-with the default value alice is `header` is missing.
+with the default value alice `header` is missing.
```

Comment:
Even with this change this sentence is still confusing - can you try to clarify it?